apps/notifications/src/AddNotifications.ts

```diff
@@ -0,0 +1,46 @@
+import { JsonFile } from '@rushstack/node-core-library';
+import * as inquirer from 'inquirer';
+import * as path from 'path';
+import { IAnswers, INotificationJson } from './configurations';
+
+export class AddNotifications {
+  public constructor() {}
+
+  public async addNotification(): Promise<void> {
+    await inquirer
+      .prompt([
+        {
+          type: 'input',
+          input: '',
+
+          name: 'message',
+          message: 'Provide the full text of the announcement to be displayed'
+        }
+      ])
+      .then((answers: IAnswers) => {
+        const pathToJson: string = path.join(
+          process.cwd(),
+          '..',
+          '..',
+          'common/config/notifications/notifications.json'
+        );
```

Comment:
This most likely needs to be a CLI argument, unless we assume we are in a rush repository and find rush.json to derive the location of the config file.